tools/accuracy_checker/accuracy_checker/evaluators/model_evaluator.py

```diff
@@ -196,6 +199,37 @@ def get_processing_info(config):
             dataset_config['name']
         )
 
+    def send_processing_info(self, sender):
+        if not sender:
+            return
+        launcher_config = self.config['launchers'][0]
+        dataset_config = self.config['datasets'][0]
+        sender.send_event('ac', 'platform', platform.system)
```

Comment:
Why not to use `send_telemetry_event` as in other cases?

Reply:
Thanks, fixed